### PR TITLE
Changed the text colour on login page in dark mode to white

### LINKED ISSUE #652: Text color change in Dark mode
### What would you like to share?

In the dark mode of the login page,the text is not visible sue to black color,
I would like to fix this under gssoc'23

### Additional information

image

### PR DESCRIPTION
**Pull request by shruuti321**
Closes #652 
I have succesfully changed the text colour on login page in dark mode to white
image

### CODE REVIEW

**Comment by shruuti321** [comment]
@narayan954  Could you pls let me know if this pull request can be merged without any conflict

**Comment by narayan954** [comment]
> @narayan954 Could you pls let me know if this pull request can be merged without any conflict

please follow up in the issue.

**Comment by shruuti321** [comment]
> > @narayan954 Could you pls let me know if this pull request can be merged without any conflict
> 
> please follow up in the issue conversation

okay, I have fixed the issue, now just waiting for the review

File: src/index.css [hunk 1]
```diff
@@ -20,6 +20,7 @@
   --bg-color: #1c2128;
   --text-primary: white;
   --text-secondary: black; /* TODO: Fix */
+  color:white;
```
Comment (narayan954) [inline src/index.css:23]: Also, this is not a proper way to add the colour. You should've made use of variables defined around

**Review by narayan954 — CHANGES_REQUESTED** [review]
changes requested, also add the fixes requested on issue comments

**Pull request closed without merging**